src/diffusers/loaders.py

```diff
@@ -1109,8 +1187,24 @@ def _remove_text_encoder_monkey_patch_classmethod(cls, text_encoder):
                 attn_module.v_proj = attn_module.v_proj.regular_linear_layer
                 attn_module.out_proj = attn_module.out_proj.regular_linear_layer
 
+        if getattr(text_encoder, "aux_state_dict_populated", False):
+            for _, aux_module in text_encoder_aux_modules(text_encoder):
+                if isinstance(aux_module.fc1, PatchedLoraProjection):
+                    aux_module.fc1 = aux_module.fc1.regular_linear_layer
+                    aux_module.fc2 = aux_module.fc2.regular_linear_layer
+
+            text_encoder.aux_state_dict_populated = False
+
     @classmethod
-    def _modify_text_encoder(cls, text_encoder, lora_scale=1, network_alpha=None, rank=4, dtype=None):
+    def _modify_text_encoder(
+        cls,
+        text_encoder,
+        lora_scale=1,
+        network_alpha=None,
+        rank=4,
+        dtype=None,
+        patch_aux=False,
```

Comment:
if there are other modules that we want to be able to optionally provide lora layers on, I think we should make the flag explicit about which modules it's adding. `aux` is pretty vague and it seems like we're just patching the mlp. So maybe we can call the flag `modify_mlp`

Reply:
There can be more in the future. So, we thought this is the best course of action.